fix: Camera Rehaul (#1299)
appsmithorg/appsmith-docs: website/docs/reference/widgets/camera.md

# Camera

This page provides instructions on how to use the camera widget to capture images and videos.

<figure>
  <img src="/img/cam-image.png" style= {{width:"700px", height:"auto"}} alt="Camera widget"/>
  <figcaption align = "center"><i>Using the Camera widget</i></figcaption>
</figure>

## Upload media to S3

> Let's see how to capture an image or video and upload it to Amazon S3.

Reviewer: ```suggestion
This guide shows you how to capture and upload an image or video to Amazon S3.
```

1. Connect to [Amazon S3](/reference/datasources/querying-amazon-s3) datasource.
2. Click the **+** icon next to the **Queries/JS** and choose your S3 datasource.

> 3. Select the method **Create a new file** from the Commands drop-down.

Reviewer: ```suggestion
3. Select **Create a new file** from the **Commands** list.
```

> 4. Specify the necessary parameters, including the Bucket Name and File Data Type. When uploading data from the Camera widget, select the `base64` option from File Data Type.

Reviewer: ```suggestion
4. Specify the necessary parameters, such as the **Bucket Name** and **File Data Type**. Select the `base64` option from **File Data Type** when uploading data from the Camera widget.
```

5. In the content box, add the data you want to upload. For instance, if you want to upload an image, use: 

 ```js
  //For image
  {"data": {{Camera1.imageDataURL}}}
  ```
To upload the video, use:
 ```js
  //For video
  {"data": {{Camera1.videoDataURL}}}
  ```

<figure>
  <img src="/img/cam-to-s3.png" style= {{width:"700px", height:"auto"}} alt="Upload media to S3"/>
  <figcaption align = "center"><i>Upload media to S3</i></figcaption>
</figure>

> 5. Set the **onImageCapture** or **onVideoSave** event to run the query. 

Reviewer: ```suggestion
5. Set the **onImageCapture** or **onVideoSave** event to run the query. When you capture an image or video using the Camera widget, it gets uploaded to the S3 bucket.
```

> When you capture and save an image or video using the camera widget, the file is automatically uploaded to the S3 bucket.

Reviewer: ```suggestion
```
